packages/app/src/files-manager/components/duplicated-files-modal/duplicated-files-modal.component.tsx

```diff
@@ -78,7 +78,6 @@ const DuplicatedFilesListItemText = styled(Text)`
 
 const DuplicatedFilesModal: FunctionComponent = ({ ...props }) => {
   const filesManagerError = useSelector(getFilesManagerError)
-
```

Comment:
What happened here : )?

Reply:
I think it looks better :D Do you think differently? :D

Reply:
i just though that it was some IDE auto correction shenanigans, if you did this on purpose that seems okay : D